cpp/CMakeLists.txt

```diff
@@ -35,14 +35,23 @@ rapids_cmake_write_version_file(include/kvikio/version_config.hpp)
 # Set a default build type if none was specified
 rapids_cmake_build_type(Release)
 
-# build options
-option(KvikIO_REMOTE_SUPPORT "Configure CMake to build with remote IO support" ON)
+# ##################################################################################################
```

Comment:
These comment blocks are super long and the lengths of the `---` aren't the same length as the `###`. Can we keep this to ~72 chars instead of ~100?

Reply:
The 100 width of the comment blocks are forced by the cmake style, which adds `=` until the width is 100. I have fixed the `----` length to match. 
I think we need a style update  in a separate PR.